command/agent_monitor.go

```diff
@@ -0,0 +1,135 @@
+package command
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/mitchellh/cli"
+)
+
+type MonitorCommand struct {
+	Meta
+}
+
+func (c *MonitorCommand) Help() string {
+	helpText := `
+Usage: nomad monitor [options]
+
+	Shows recent log messages of a nomad agent, and attaches to the agent,
+	outputting log messagse as they occur in real time. The monitor lets you
+	listen for log levels that may be filtered out of the Nomad agent. For
+	example your agent may only be logging at INFO level, but with the monitor
+	command you can set -log-level DEBUG
+
+General Options:
+
+	` + generalOptionsUsage() + `
+	
+Monitor Specific Options:
+
+  -log-level <level>
+    Sets the log level to monitor (default: INFO)
+
+  -node-id <node-id>
+    Sets the specific node to monitor
+
+  -log-json <true|false>
```

Comment:
```suggestion
  -json <true|false>
```

This matches a bunch of our other CLI commands that have json formats as well. The difference here being this will be ndjson vs a single json object like other apis, but I think that's ok.